### PR TITLE
Add README.md for glue submission formatter script

### PR DESCRIPTION
**Pull request by jeswan**

### CODE REVIEW

File: jiant/scripts/postproc/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# NLP Benchmark Submission Formatter Scripts
+
+## GLUE Benchmark Submission Formatter
+
+Jiant supports [GLUE Benchmark](https://gluebenchmark.com/) test prediction generation in the submission format. To generate test predictions, use the `--write_test_preds` flag in [runscript.py](https://github.com/jiant-dev/jiant/blob/master/jiant/proj/main/runscript.py) when running your workflow. This will generate a `test_preds.p` file in the specified output directory. To convert `test_preds.p` to the required GLUE submission format, run the following:
```
Comment (sleepinyourhat) [inline jiant/scripts/postproc/README.md:5]: nit, `jiant`, because we're maybe stuck with this stupid convention.

Comment (sleepinyourhat) [reply jiant/scripts/postproc/README.md:5]: "test prediction generation in the submission format" => "generating submission files".

**Comment by zphang** [comment]
Can we move the documentation to an examples or documentation folder?

**Comment by jeswan** [comment]
@zphang I think that is a fair point. My plan is to modify this PR and split it into two parts:

1. Document the GLUE submission formatter script within the CLI tool  (`--help`)
2. Add a brief description of the script in a central documentation location so that users know it exists

**Pull request merged**